docs/01-tooling/03-fast-and-isolated-JS-unit-tests.md

```diff
@@ -2,10 +2,159 @@
 
```

Comment:
Consider adding a section before the intro that summarizes the article. Not this wording but like: we were using slow QUnit integration tests but but switched to fast Node.js unit tests.

Reply:
I've added the conclusions section like in the other articles so the structure should be familiar